api/src/main/java/org/openmrs/module/openhmis/commons/web/PrivilegeConstantsCompatibility.java

```diff
@@ -0,0 +1,189 @@
+package org.openmrs.module.openhmis.commons.web;
+
+import org.openmrs.Privilege;
+import org.openmrs.api.context.Context;
+
+/**
+ * Gets the correct default view visit privilege needed.
+ * @return A set containing the default set of privileges.
+ */
+public class PrivilegeConstantsCompatibility {
+	/*Cashier module privilege constants*/
+	public static final String GET_PROVIDERS = "Get Providers";
+	public static final String VIEW_PROVIDERS = "View Providers";
+	public static final String GET_VISITS = "Get Visits";
+	public static final String VIEW_VISITS = "View Visits";
+	public static final String ADD_ENCOUNTERS = "Add Encounters";
+	public static final String ADD_VISITS = "Add Visits";
+	public static final String EDIT_ENCOUNTERS = "Edit Encounters";
+	public static final String EDIT_PATIENTS = "Edit Patients";
+	public static final String EDIT_VISITS = "Edit Visits";
+	public static final String DASHBOARD_SUMMARY = "Patient Dashboard - View Patient Summary";
+	public static final String DASHBOARD_DEMOGRAPHICS = "Patient Dashboard - View Demographics Section";
+	public static final String DASHBOARD_OVERVIEW = "Patient Dashboard - View Overview Section";
+	public static final String DASHBOARD_VISITS = "Patient Dashboard - View Visits Section";
+	public static final String VIEW_ADMIN_FUNCTIONS = "View Administration Functions";
+	public static final String GET_ADMIN_FUNCTIONS = "Get Administration Functions";
+	public static final String VIEW_ENCOUNTERS = "View Encounters";
+	public static final String GET_ENCOUNTERS = "Get Encounters";
+	public static final String VIEW_NAVIGATION_MENU = "View Navigation Menu";
+	public static final String GET_NAVIGATION_MENU = "Get Navigation Menu";
+	public static final String VIEW_OBS = "View Observations";
+	public static final String GET_OBS = "Get Observations";
+	public static final String VIEW_PATIENTS = "View Patients";
+	public static final String GET_PATIENTS = "Get Patients";
+	
+	/*Inventory module privilege constants*/
+	public static final String GET_ROLES = "Get Roles";
+	public static final String VIEW_ROLES = "View Roles";
+	public static final String GET_USERS = "Get Users";
+	public static final String VIEW_USERS = "View Users";
+	public static final String GET_CONCEPTS = "Get Concepts";
+	public static final String VIEW_CONCEPTS = "View Concepts";
+	public static final String GET_LOCATIONS = "Get Locations";
+	public static final String VIEW_LOCATIONS = "View Locations";
+	public static final String EDIT_PATIENT_IDENTIFIERS = "Edit Patient Identifiers";
+	
+	public String checkPrivilege(String para1, String para2) {

```

Comment:
Update these parameter names to be useful. I shouldn't need to read the code to figure out what a parameter is used for.